### DISCUSSION TITLE
will nginxproxymanager set HTTP_X_REAL_IP automaticly?

### DISCUSSION THREAD

**Discussion by q16marvin** [NginxProxyManager/nginx-proxy-manager#2596] [Q&A]
Hi,

im littly confused, doese nginxproxymanager set this header automaticly?

Thx!

Erik

**Comment by weasam** [comment]
yes. 
If you are using docker like what I am using
docker exec -it silly_kirch bash 
all config files are in /etc/nginx
set_real_ip  and proxy_set_header  has the enable get and forward the real IP to the server.

**but but I'm stuck I need others help**
I'm using Nginx Proxy Manager in OpenWRT, My server logs getting all IP is from Nginx proxy manager 172.10.1.1 I read another issue is the NAT or firewall problem of OpenWRT.
